files/firstLaunch.js

```diff
@@ -341,11 +340,9 @@ module.exports = async () => {
 
                 // unzip librespeed-cli
                 console.log(`Unzipping librespeed-cli...`)
-                fs.createReadStream(`${process.cwd()}/files/librespeed-cli/librespeed-cli.zip`)
-                    .pipe(unzipper.Extract({ path: `${process.cwd()}/files/librespeed-cli/` }))
 
-                    // when unzipping is done, delete zip file
-                    .on("close", () => {
+                asyncExtract(`${process.cwd()}/files/librespeed-cli/librespeed-cli.zip`, `${process.cwd()}/files/librespeed-cli/`, 'librespeed', '/files/librespeed-cli/librespeed-cli.zip')
+                    .then(() => {
                         console.log(`Finished unzipping librespeed-cli.`)
                         fs.unlinkSync(`${process.cwd()}/files/librespeed-cli/librespeed-cli.zip`)
```

Comment:
since we will remove the original zip in the asyncExtract, we don't need to remove it here